docs/BackupAndRestore.md

```diff
@@ -103,6 +103,42 @@ Use `nuobackup --type report-latest --db-name <db> --group <backup group>` to sh
 It is possible to disable backup jobs creation by setting `hotcopy.enableBackups` to `false`.
 Custom backup jobs that use the `nuobackup` script can be configured to support more complex workflows and backup strategies.
 
+The logs for each backup job can be seen using `kubectl get logs <job pod name>`. For example:
+
+```bash
+kubectl logs full-hotcopy-demo-cronjob-1621415040-vxlq8 --namespace nuodb
+```
+
+A successful hot copy will mark the job pod status as `Completed` and will report hot copy status as `completed`.
+An example logs from _full_ hot copy can be seen below.
+
+```
+Starting full backup for database demo on processes with labels 'backup cluster0 ' ...
+{
+  "beginTimestamp": "2021-05-19 09:04:06",
+  "coordinatorStartId": "1",
+  "destinationDirectory": "/var/opt/nuodb/backup/20210519T090406/tmp/full/data",
+  "endTimestamp": "2021-05-19 09:04:07",
+  "hotCopyId": "e5b8b9ba-f29f-440e-8c67-1c71cef34292",
+  "message": "Hot copy successfully completed",
+  "stage": 3,
+  "status": "completed",
+  "uri": "http://nuodb.nuodb.svc:8888/api/1/databases/hotCopyStatus?coordinatorStartId=1&hotCopyId=e5b8b9ba-f29f-440e-8c67-1c71cef34292"
+}
+/nuodb/nuobackup/demo/cluster0/latest = 2
+/nuodb/nuobackup/demo/cluster0/2 = 20210519T090406
+/nuodb/nuobackup/demo/latest = cluster0
+```
+
+The above _full_ backup will be available in all _RUNNING_ HC SMs at `/var/opt/nuodb/backup/20210519T090406` with the following content:
+
+```
+$ kubectl exec -ti --namespace nuodb \
```

Comment:
Same comment really for all examples in the guide, to include namespace in the command or leave it out, I'm ok with either as long as its consistent.